### PR TITLE
Add @no_default_encryption tag to the feature

### PR DESCRIPTION
**Pull request by sharidas**
## Description

This is to fix the transfer-ownership.feature failure for the stable10 branch.

## Related Issue

This change tries to address the issue: https://github.com/owncloud/core/issues/28625

## Motivation and Context

transfer-ownership.feature would fail if just encryption is enabled. We need to enable masterkey too to satisfy the test.

## How Has This Been Tested?

Had ran ```OC_TEST_ALT_HOME=1 OC_TEST_ENCRYPTION_ENABLED=1 OC_TEST_ENCRYPTION_MASTER_KEY_ENABLED=1``` ./run.sh in my local machine and verified.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -142,7 +142,7 @@ timestampedNode('SLAVE') {
 				rm -rf config/config.php data/*
 				./occ maintenance:install --admin-pass=admin
 				make clean-test-integration
-				make test-integration OC_TEST_ALT_HOME=1 OC_TEST_ENCRYPTION_ENABLED=1
+				make test-integration OC_TEST_ALT_HOME=1 OC_TEST_ENCRYPTION_ENABLED=1 OC_TEST_ENCRYPTION_MASTER_KEY_ENABLED=1
```
Comment (PVince81) [inline Jenkinsfile]: This kills the user key test. We need to keep a test run for user key mode too.

You probably need to change the next scenario and add `OC_TEST_ENCRYPTION_ENABLED` below instead

**Comment by PVince81** [comment]
another idea for the future: only have a single encryption related env var: `TEST_ENCRYPTION_MODE` with values "", "userkey" or "masterkey".

**Comment by PVince81** [comment]
@sharidas still WIP or ready for review ?

**Comment by sharidas** [comment]
@PVince81 I am looking at the logs of jenkins in parallel. In my local machine didn't found any issue. Tried executing transfer-ownership.feature from scratch install. The exit code is 0.

**Comment by sharidas** [comment]
Here are the commands executed and the results shared along with it:
- ```OC_TEST_ALT_HOME=1 OC_TEST_ENCRYPTION_ENABLED=1 ./run.sh features/transfer-ownership.feature``` log file here: 
[withencryptiononly.txt](https://github.com/owncloud/core/files/1211654/withencryptiononly.txt)
- ```OC_TEST_ALT_HOME=1 OC_TEST_ENCRYPTION_MASTER_KEY_ENABLED=1 ./run.sh features/transfer-ownership.feature``` log file here: 
[withmasterkey.txt](https://github.com/owncloud/core/files/1211684/withmasterkey.txt)
- ```OC_TEST_ALT_HOME=1  ./run.sh features/transfer-ownership.feature``` log file here: 
[withnoencryption.txt](https://github.com/owncloud/core/files/1211694/withnoencryption.txt)

In all the the three logs the exit code is 0.

**Review by PVince81 — APPROVED** [review]
:+1: thanks

**Comment by PVince81** [comment]
likely needs a similar fix on master

**Comment by sharidas** [comment]
@PVince81 Yup. I was thinking about the same. Will do it. Thanks for the support.

**Pull request merged**